cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/DefaultHBaseDDLExecutor.java

```diff
@@ -118,14 +120,15 @@ public void createTableIfNotExists(TableDescriptor descriptor, @Nullable byte[][
       return;
     }
 
+    boolean tableExistsFailure = false;
```

Comment:
these changes here are to prevent confusing logs such as:
```
2017-02-01 15:34:45,866 - DEBUG [main:c.c.c.d.u.h.DefaultHBaseDDLExecutor@122] - Attempting to create table 'cdap_perm:priv' if it does not exist
2017-02-01 15:34:48,125 - INFO  [main:c.c.c.d.u.h.DefaultHBaseDDLExecutor@124] - Table created 'cdap_perm:priv'
2017-02-01 15:34:48,131 - INFO  [main:c.c.c.d.u.h.DefaultHBaseDDLExecutor@139] - Table 'cdap_perm:priv' exists now. Assuming that another process concurrently created it.
```
which was coming out for every table created.